.github/workflows/compat-docker-stable.yml

```diff
@@ -1,20 +1,20 @@
-name: Compat Check Docker - Lightning@master
+name: Compat Check Docker - Lightning@stable
 
 on:
   schedule:
     - cron: "0 4 * * 1-5"  # Run daily at 4am Mon-Fri
   workflow_dispatch:
 
 concurrency:
-  group: docker_linux_x86_64-master
+  group: docker_linux_x86_64-stable
   cancel-in-progress: true
 
 jobs:
-  docker_linux_x86_64_develop:
-    name: Docker develop - Linux::x86_64
+  docker_linux_x86_64_stable:
+    name: Docker stable - Linux::x86_64
     uses: ./.github/workflows/docker_linux_x86_64.yml
     with:
-      lightning-version: master
-      pennylane-version: master
+      lightning-version: v0.37.0
+      pennylane-version: v0.37.0
```

Comment:
Isn't the stable tag applicable here?

Reply:
No, we do not maintain a `stable` tag. We could use `latest_release` for `lightning-version` but PennyLane does not have an equivalent tag it appears.